components/BookstoreItem.vue

```diff
@@ -17,10 +17,10 @@
         v-text="bookName"
       />
 
-      <div
+      <NuxtLink
+        :to="bookInfo.authorPageRoute.value"
         class="h-4 laptop:h-5 mt-0.5 text-xs laptop:text-sm text-[#9B9B9B] line-clamp-1"
-        v-text="authorName"
-      />
+      >{{ authorName }}</NuxtLink>
```

Comment:
<img width="394" height="179" alt="Image" src="https://github.com/user-attachments/assets/b94e778d-d178-4870-8c8d-14a763183db7" />

```suggestion
      <NuxtLink
        :to="bookInfo.authorPageRoute.value"
class="inline-block h-4 laptop:h-5 mt-0.5 text-xs laptop:text-sm text-[#9B9B9B] line-clamp-1 hover:text-theme-500 hover:underline"
      >{{ authorName }}</NuxtLink>
```